### ISSUE TITLE
Fine tune 更多的层？

### ISSUE THREAD

**Issue by digitalized-hfut** [hzy46/Deep-Learning-21-Examples#38]
想请教一下   如何fine tune 更多的层   还有 如过换成其他的网络呢？
--checkpoint_path=satellite/pretrained/inception_v3.ckpt \  这里我知道是加载之前训练好的模型权值
--checkpoint_exclude_scopes=InceptionV3/Logits,InceptionV3/AuxLogits \   这里是不加载的层信息，但是这些信息是怎么来的，或者里面是说明代码，不太清楚。
--trainable_scopes=InceptionV3/Logits,InceptionV3/AuxLogits \如果换成其他网络是不是把“Inception”换成“Resnet”？？？？
如果想fine tune 之前更多的层呢？？
谢谢

**Comment by hzy46** [comment]
可以从Slim里定义好网络，然后打印当前计算图所有Tensor的名字，这样就知道它们的名字了：https://blog.csdn.net/shwan_ma/article/details/78879620

**Comment by digitalized-hfut** [comment]
@hzy46    非常感谢
